src/lib/objects/schemas.ts

```diff
@@ -17,3 +17,27 @@ export const UserSchema = z.object({
 	avatar: z.string().optional(),
 })
 export type User = z.infer<typeof UserSchema>
+
+export const TransactionSchema = z.object({
+	token: z.object({
+		amount: z.string(),
+		symbol: z.string(),
+		decimals: z.number().int().positive(),
+	}),
+	to: AddressSchema,
+	from: AddressSchema,
+	fee: z.object({
+		amount: z.string(),
+		symbol: z.string(),
+		decimals: z.number().int().positive(),
+	}),
```

Comment:
Let's keep it as is right now, but later on we should think about whether we don't want to use `token address` instead of `symbol` as symbols can be ambiguous (although they should not)